widget/src/components/UserInput.tsx

```diff
@@ -6,25 +6,25 @@
  * 2. All derivative works must include clear attribution to the original creator and software, Hexastack and Hexabot, in a prominent location (e.g., in the software's "About" section, documentation, and README file).
  */
 
-import React, { useEffect, useRef, useState } from 'react';
-
-import { useTranslation } from '../hooks/useTranslation';
-import { useChat } from '../providers/ChatProvider';
-import { useColors } from '../providers/ColorProvider';
-import { useSettings } from '../providers/SettingsProvider';
-import { TOutgoingMessageType } from '../types/message.types';
-import { OutgoingMessageState } from '../types/state.types';
-
-import EmojiButton from './buttons/EmojiButton';
-import FileButton from './buttons/FileButton';
-import LocationButton from './buttons/LocationButton';
-import MenuButton from './buttons/MenuButton';
-import SendButton from './buttons/SendButton';
-import CloseIcon from './icons/CloseIcon';
-import FileInputIcon from './icons/FileInputIcon';
-import Suggestions from './Suggestions';
-
-import './UserInput.scss';
+import React, { useEffect, useRef, useState } from "react";
+
+import { useTranslation } from "../hooks/useTranslation";
+import { useChat } from "../providers/ChatProvider";
+import { useColors } from "../providers/ColorProvider";
+import { useSettings } from "../providers/SettingsProvider";
+import { TOutgoingMessageType } from "../types/message.types";
+import { OutgoingMessageState } from "../types/state.types";
+
+import EmojiButton from "./buttons/EmojiButton";
+import FileButton from "./buttons/FileButton";
+import LocationButton from "./buttons/LocationButton";
+import MenuButton from "./buttons/MenuButton";
+import SendButton from "./buttons/SendButton";
+import CloseIcon from "./icons/CloseIcon";
+import FileInputIcon from "./icons/FileInputIcon";
+import Suggestions from "./Suggestions";
+
+import "./UserInput.scss";
```

Comment:
@yassinedorbozgithub Can we fix the linter to be consistent with frontend ? Maybe a seperate PR to lint:fix everything ?

Reply:
Sure, happy to do that like mentioned in a separate PR